php-fpm/Dockerfile-70

```diff
@@ -23,6 +23,10 @@ RUN curl -L -o /tmp/memcached.tar.gz "https://github.com/php-memcached-dev/php-m
     && docker-php-ext-install memcached \
     && rm /tmp/memcached.tar.gz
 
+# Install mongodb driver
+RUN pecl install mongodb
+RUN echo "extension=mongodb.so" > /usr/local/etc/php/conf.d/mongo.ini
```

Comment:
Can you try adding `extension=mongodb.so` to [laravel.ini](https://github.com/LaraDock/laradock/blob/master/php-fpm/laravel.ini) and removing it from here to see if it still work; and if it does please add another commit with this change.